db/migrate/20200729042607_create_discourse_voting_vote_counters.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class CreateDiscourseVotingVoteCounters < ActiveRecord::Migration[6.0]
+  def up
+    create_table :discourse_voting_vote_counters do |t|
```

Comment:
`discourse_voting_vote_counters` doesn't seem clear to me as to what data the table holds. Perhaps `discourse_voting_topic_vote_count`?

Reply:
I'm also wondering if this table is necessary since we can calculate the vote counts from the `discourse_voting_votes` table.

Reply:
Yeah fair point, I think it might be useful because that cache would speed up topics page (we will not need to join votes and count). Before that table, it was using topics_custom_field to collect count, so I was thinking that it makes sense to continue